src/Mod/Path/PathScripts/PostUtilsExport.py

```diff
@@ -71,44 +71,44 @@
 def export_common(values, objectslist, filename):
     """Do the common parts of postprocessing the objects in objectslist to filename."""
     #
+    nl = "\n"
+
     for obj in objectslist:
         if not hasattr(obj, "Path"):
             print(
-                "The object "
-                + obj.Name
-                + " is not a path. Please select only path and Compounds."
+                f"The object {obj.Name} is not a path. Please select only path and Compounds."
             )
             return None
 
     # for obj in objectslist:
     #    print(obj.Name)
 
-    print("PostProcessor:  " + values["POSTPROCESSOR_FILE_NAME"] + " postprocessing...")
+    print(f'PostProcessor:  {values["POSTPROCESSOR_FILE_NAME"]} postprocessing...')
```

Comment:
```suggestion
    print(f"PostProcessor:  {values["POSTPROCESSOR_FILE_NAME"]} postprocessing...")
```
Some places you use double quotes, other places single quotes. be aware

Reply:
I use single quotes when there are double quotes inside the "f-string".  Both of the examples you pointed out are not happy with double quotes surrounding the whole f-string.  It is slightly less of a pain to switch to using single quotes to surround the string instead of using "\" in front of the double quotes inside the string, I think.

Reply:
That should have been a backslash in the "" :-).